engine/src/w32script.cpp

```diff
@@ -596,10 +596,9 @@ void MCWindowsActiveScriptEnvironment::Finalize(void)
 void MCWindowsActiveScriptEnvironment::Run(MCStringRef p_script, MCStringRef& r_out)
 {
 	LPOLESTR t_ole_script;
```

Comment:
I'm pretty sure LPOLESTR is the same as LPWSTR - so I think we can use MCAutoStringRefAsWString here. I think the reason that LPOLESTR is used in this case (in the Win32 API) is that the type of strings used in OLE were always Unicode - whereas other Win32 APIs would be Ansi or Unicode.